docs/sources/vendor/Aruba/silverpeak.md

```diff
@@ -0,0 +1,44 @@
+# Clearpass
+
+## Key facts
+
+* Requires vendor product by source configuration
+* Legacy BSD Format default port 514
+
+## Links
+
+| Ref            | Link                                                                                                    |
+|----------------|---------------------------------------------------------------------------------------------------------|
+
+## Sourcetypes
+
+| sourcetype       | notes                                                                                                   |
+|------------------|---------------------------------------------------------------------------------------------------------|
+| aruba:silverpeak | Dynamically  Created                                                                                    |
```

Comment:
We can remove `Dynamically  Created` from this doc

Reply:
Done